src/libs/ReportUtils.ts

```diff
@@ -1850,10 +1876,74 @@ function getPolicyExpenseChatName(report: OnyxEntry<Report>, policy: OnyxEntry<P
     return reportOwnerDisplayName;
 }
 
+/**
+ * Given a report field, check if the field is for the report title.
+ */
+function isReportFieldOfTypeTitle(reportField: OnyxEntry<PolicyReportField>): boolean {
+    return reportField?.type === 'formula' && reportField?.fieldID === CONST.REPORT_FIELD_TITLE_FIELD_ID;
```

Comment:
Should Custom report name field necessarily be a formula?

We have an issue here #49077. The bug here is if we set the custom report name as some text string instead of a formula, backend returns the type of the title field as `text`. Then this check fails and `Title` field shows up in the report though it should not show up here as per this code

https://github.com/Expensify/App/blob/13549575f6c908282e66a95f19406cb909c0a92f/src/components/ReportActionItem/MoneyReportView.tsx#L108-L109

Can you please help here?

@thienlnam @allroundexperts @jjcoffee